scripts/convert_yaml_to_sdf.py

```diff
@@ -0,0 +1,207 @@
+            shape_items_pose = fill_pose(shape_items)
+        elif 'pose' in shape_items[compound_type]:
+            # If 'pose' is in compound_type then check if coordinates are in that level and overwrite default 0
+            shape_items_pose = fill_pose(shape_items[compound_type])
+
+        if compound_type == "cylinder":
+
+            sdf_dict.update({'collision name="{}"'.format(shape_name):
+                                 {'pose': shape_items_pose,
+                                  'geometry': {'cylinder': {'radius': shape_items['cylinder']['radius'],
+                                                            'length': shape_items['cylinder']['height']}}},
+                             'visual name="{} visual"'.format(shape_name):
+                                 {'pose': shape_items_pose,
+                                  'geometry': {'cylinder': {'radius': shape_items['cylinder']['radius'],
+                                                            'length': shape_items['cylinder']['height']}}}
+                             })
+            if object_color != [0, 0, 0, 1]:
+                sdf_dict['visual name="{} visual"'.format(shape_name)] = \
+                    {'material': {'ambient': str(object_color).replace("[", "").replace("]", "").replace(",", "")}}
+
+        elif compound_type == "box":
+
+            shape_items_size = [shape_items['box']['size']['x'],
+                                shape_items['box']['size']['y'],
+                                shape_items['box']['size']['z']]
+            shape_items_size = str(shape_items_size).replace("[", "").replace("]", "").replace(",", "")
+
+            sdf_dict.update({'collision name="{}"'.format(shape_name):
+                                 {'pose': shape_items_pose,
+                                  'geometry': {'box': {'size': shape_items_size}}},
+                             'visual name="{} visual"'.format(shape_name):
+                                 {'pose': shape_items_pose,
+                                  'geometry': {'box': {'size': shape_items_size}}}
+                             })
+            if object_color != [0, 0, 0, 1]:
+                sdf_dict['visual name="{} visual"'.format(shape_name)] = \
+                    {'material': {'ambient': str(object_color).replace("[", "").replace("]", "").replace(",", "")}}
+
+    return sdf_dict
+
+
+def main():
+    # if len(sys.argv) <= 1:
+    #     print "Please provide a file as input"
+    #     sys.exit(1)
+
+    # Create an empty dictionary in SDF format, which can later on be exported to XML after filling below.
+    sdf_dict = {}
+
+    # First determine type of model.yaml
+    # Raw input can be replaced by fancier read_option in create-model.py
+    # file_type = raw_input("File type ('world' or 'object'): ")
+    # model_name = raw_input("Model name: ")
+
+    # Temporary replacements of user input
+    file_type = "object"
+
+    with open("model.yaml", 'r') as stream:
```

Comment:
"model.yaml" is a local path. It might be more usefull to create an input with the name of the model and then search for that model in the ed_object_models/models folder. This can also help you since you will need the model name later anyway.